musicbrainzngs/mbxml.py

```diff
@@ -505,7 +505,9 @@ def parse_disc(disc):
     result = {}
     attribs = ["id"]
     elements = ["sectors"]
-    inner_els = {"release-list": parse_release_list}
+    inner_els = {"release-list": parse_release_list,
+                 "offset-list": parse_offset_list
```

Comment:
Thanks, I forgot that I manually added info about the returned data. (fixed with next commit)